demos/projects/ESPRESSIF/adu/main/azure_iot_freertos_esp32_pnp_model.h

```diff
@@ -74,7 +74,7 @@ uint32_t ulSampleCreateReportedPropertiesUpdate( uint8_t * pucPropertiesData,
 
 // Azure Device Update
 extern az_iot_adu_client_update_request xUpdateRequest;
-extern az_iot_adu_client_device_information xDeviceInformation;
+extern az_iot_adu_client_device_properties xDeviceInformation;
```

Comment:
ugh I wish this was named something else with all the overloading of the term "device properties". But it has the ADU prefix stuff so I think it's okay

Reply:
Yeah, I rather keep it similar to the json property defined by the ADU team.
And as you mentioned, these properties and structures have "ADU" as prefix, so it is still specific to ADU.